src/utils/common.ts

```diff
@@ -0,0 +1,6 @@
+export const stripArrayTypeIfPresent = (typeString: string) => {
```

Comment:
Nit: Would be nice to have a JSDoc block here as well

Reply:
done